swagger-spec/taxonomies/list.yaml

```diff
@@ -1 +1,175 @@
+              parents:
+              - text: Teacher Education and Professional Development
+                id: 584240d954be81056ceca8fd
+              child_count: 0
+            type: taxonomies
+            id: 584240d854be81056ceca83c
+          - links:
+              self: https://api.osf.io/v2/taxonomies/584240d954be81056ceca83d/
+              parents:
+              - https://api.osf.io/v2/taxonomies/584240d954be81056ceca938/
+            attributes:
+              text: Other Educational Administration and Supervision
+              parents:
+              - text: Educational Administration and Supervision
+                id: 584240d954be81056ceca938
+              child_count: 0
+            type: taxonomies
+            id: 584240d954be81056ceca83d
+          - links:
+              self: https://api.osf.io/v2/taxonomies/584240d954be81056ceca83e/
+              parents:
+              - https://api.osf.io/v2/taxonomies/584240d954be81056ceca953/
+            attributes:
+              text: Russian Linguistics
+              parents:
+              - text: Slavic Languages and Societies
+                id: 584240d954be81056ceca953
+              child_count: 0
+            type: taxonomies
+            id: 584240d954be81056ceca83e
+          - links:
+              self: https://api.osf.io/v2/taxonomies/584240d954be81056ceca83f/
+              parents:
+              - https://api.osf.io/v2/taxonomies/584240d954be81056cecaa24/
+            attributes:
+              text: Medicinal Chemistry and Pharmaceutics
+              parents:
+              - text: Pharmacology, Toxicology and Environmental Health
+                id: 584240d954be81056cecaa24
+              child_count: 0
+            type: taxonomies
+            id: 584240d954be81056ceca83f
+          - links:
+              self: https://api.osf.io/v2/taxonomies/584240d954be81056ceca840/
+              parents:
+              - https://api.osf.io/v2/taxonomies/584240da54be81056cecac49/
+            attributes:
+              text: Poetry
+              parents:
+              - text: Creative Writing
+                id: 584240da54be81056cecac49
+              child_count: 0
+            type: taxonomies
+            id: 584240d954be81056ceca840
+          - links:
+              self: https://api.osf.io/v2/taxonomies/584240d954be81056ceca841/
+              parents:
+              - https://api.osf.io/v2/taxonomies/584240da54be81056cecab8f/
+            attributes:
+              text: Public Economics
+              parents:
+              - text: Economics
+                id: 584240da54be81056cecab8f
+              child_count: 0
+            type: taxonomies
+            id: 584240d954be81056ceca841
+          links:
+            first:
+            last: https://api.osf.io/v2/taxonomies/?page=122
+            prev:
+            next: https://api.osf.io/v2/taxonomies/?page=2
+            meta:
+            total: 1217
```

Comment:
Can you change the total here to match how many taxonomies you've included in this example? Just so it seems more accurate.